xml/ha_storage_protection.xml

```diff
@@ -781,18 +780,44 @@ Timeout (msgwait)  : 10
        would be appropriate if the <literal>msgwait</literal> timeout value for
        SBD was set to <literal>30</literal> seconds.</para>
      </callout>
-     <callout arearefs="co.ha.sbd.st.rsc">
-      <para>
-       The &stonith; resource to use for SBD.</para>
-     </callout>
-     <callout arearefs="co.ha.sbd.pm_delay_max">
+   </calloutlist>
+  </step>
+  <step>
+   <para>For a two-node cluster, decide if you want predictable or random delays. For
+   other cluster setups you do not need to set this parameter.</para>
+   <variablelist>
+    <varlistentry>
+     <term>Predictable Static Delays</term>
+     <listitem>
+      <para>This parameter enables a static delay before executing &stonith; actions.
+      It ensures that the nodes do not fence each other if separate fencing resources
+      and different delay values are being used. The targeted node will
+      loose in a <quote>fencing race</quote>.
+      The parameter can be used to <quote>mark</quote> a specific node to survive
+      in case of a split-brain scenario in a two-node cluster.
```

Comment:
How about:

The parameter can be used to <quote>mark</quote> a specific node being targeted with a longer fencing delay, so that it wins fencing match in case of split-brain in case of a split-brain scenario in a two-node cluster.

Reply:
Yes, I've added your sentence.